week1/04.py

```diff
@@ -0,0 +1,4 @@
+Dictionary = {0: 10, 1: 20}
+Dictionary[2] = 30
+
+print(Dictionary)
```

Comment:
`Dictionary` is a reserved word in Python for the Datatype, and probably a bad variable name to choose :)